ml-agents/mlagents/trainers/trainer_util.py

```diff
@@ -97,11 +96,7 @@ def initialize_trainer(
         trainer_parameters.update(trainer_config[_brain_key])
 
     trainer = None
-    if trainer_parameters["trainer"] == "offline_bc":
```

Comment:
Can you leave this in and raise an error message with upgrade instructions, at least for one release?

Reply:
Added exception message

Reply:
nit: I'd say "removed", not "deprecated". To me, deprecated means "going to be removed in the future, but still around for now"

Reply:
changed to removed